src/SecureHeaders.php

```diff
@@ -1246,30 +1189,32 @@ private function deconstructHeaderValue(
         return $attributes;
     }
 
-    private function validateHeaders()
+    private function validateHeaders(HeaderBag $headers)
```

Comment:
I don't think this works (CSP needs to be special cased in `Header` as the attribute decomposition algorithm differs)